services/device-registry-base/src/main/java/org/eclipse/hono/deviceregistry/service/device/AbstractRegistrationService.java

```diff
@@ -55,8 +75,108 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(AbstractRegistrationService.class);
 
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
why do we need this?

Reply:
We don't. I didn't know about the `mapTo()` method in `JsonObject` which I learnt just today.